stable/kommander/requirements.lock

```diff
@@ -5,5 +5,8 @@ dependencies:
 - name: kommander-cluster-lifecycle
   repository: https://mesosphere.github.io/kommander-cluster-lifecycle/charts
   version: v0.1.9
-digest: sha256:553d3c5c6ae5dda10093e5149170e92a0313958d64718add8716f514e912e211
-generated: "2019-10-25T10:30:23.462935-07:00"
+- name: kommander-thanos
+  repository: https://branden.github.io/charts/stable
```

Comment:
this should be mesosphere repo :/. That tells me we need two PRs one with kommander-thanos and another using it.